css/styles.css

```diff
@@ -1,3 +1,27 @@
+.wcpdf-number-tools-sections ul {
+	height: 3em;
+}
+
+.wcpdf-number-tools-sections ul li {
+	float: left;
+	margin-right: 10px;
+}
+
+.wcpdf-number-tools-sections ul li a {
+	text-decoration: none;
+	display: inline-block;
+	padding: 0.8em 1em;
+	color: #50575e;
+	border:  1px solid #eaeaea;
+	box-sizing: border-box;
+}
+
+.wcpdf-number-tools-sections ul li a.active {
+	border: 2px solid #51266b;
+	padding: calc(0.8em - 1px) calc(1em - 1px);
+	color: black;
+}
```

Comment:
I think we should handle this part in the main plugin (with a more generic class), so that any addon with sections is rendered the same way.

Reply:
Solved in commit: https://github.com/wpovernight/woocommerce-pdf-ips-number-tools/pull/13/commits/54b017a17ffcaf5343a684ee84c4f3a4e3a52dc6